cdc/processor/sinkmanager/manager_test.go

```diff
@@ -401,11 +401,6 @@ func TestGetTableStats(t *testing.T) {
 		return checkpointTS.ResolvedMark() == 3
 	}, 5*time.Second, 10*time.Millisecond)
 
-	cleanPos := engine.Position{
```

Comment:
Please keep this assertion. Otherwise, this test would useless.

Reply:
`GetTableStats` won't drive GC now. So we can remove this from the case.

Reply:
So we need to change or add a test case to cover this logic.

Reply:
If you don't have time to add it. At least we need to delete this test. Because it's just for GC logic.

Reply:
Let's delete the case for now.